### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by YuriiBobeshko**

### CODE REVIEW

File: ormconfig.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default {
+  type: 'postgres',
+  host: 'localhost',
+  port: 6000,
+  username: 'postgres',
+  driver: 'postgres',
+  password: 'idazov',
```
Comment (boale) [inline ormconfig.ts]: should it be here?

File: ormconfig.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  database: 'nodeJs',
+  entities: [__dirname + 'src/**/*.entity.{ts,js}'],
```
Comment (boale) [inline ormconfig.ts]: use `path.join(__dirname, 'src/**/*.entity.{ts,js}')` for better path handling

File: ormconfig.ts (hunk 1 above)
Comment (YuriiBobeshko) [reply ormconfig.ts]: fixed

File: ormconfig.ts (hunk 2 above)
Comment (YuriiBobeshko) [reply ormconfig.ts]: fixed

File: src/main.ts [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { IoAdapter } from '@nestjs/platform-socket.io';
+import { WsAdapter } from '@nestjs/platform-ws';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.useWebSocketAdapter(new WsAdapter(app));
```
Comment (boale) [inline src/main.ts]: Have you managed to make it alive?

File: src/seeds/seeder.module.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+@Module({
+  imports: [],
+  providers: [SeederService, ...REPOS],
+  exports: [SeederService, ...REPOS],
+})
+export class SeederModule implements OnApplicationBootstrap {
+  constructor(private readonly seederService: SeederService) {}
+
+  async onApplicationBootstrap(): Promise<any> {
```
Comment (boale) [inline src/seeds/seeder.module.ts:16]: do we need to seed this data every time the app bootstraps?

File: src/seeds/seeder.repository.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { Users } from '../users/users.model';
+import { NewUser } from '../types/users';
+
+@Injectable()
+export class SeederRepository {
+  constructor(
+    @Inject(Users.name)
```
Comment (boale) [inline src/seeds/seeder.repository.ts:8]: 👍

File: src/seeds/seeder.module.ts (hunk 4 above)
Comment (YuriiBobeshko) [reply src/seeds/seeder.module.ts:16]: it will work only if we don't have any data in table

File: src/main.ts (hunk 3 above)
Comment (YuriiBobeshko) [reply src/main.ts]: Unfortunately I haven't managed
but I will try later

File: src/seeds/seeder.repository.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    private readonly userModel: typeof Users,
+  ) {}
+
+  checkDataUsers(id: number): Promise<boolean> {
```
Comment (boale) [inline src/seeds/seeder.repository.ts]: can be renamed to isUserDataExist or something like that.

File: src/seeds/seeder.repository.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    return this.userModel
+      .findOne({
+        where: { id },
+      })
+      .then((res) => res !== null);
+  }
+
+  createUsers(users: NewUser[]): Promise<NewUser[]> {
```
Comment (boale) [inline src/seeds/seeder.repository.ts:20]: as all methods in the service is already scoped by class itself, no need to add `Users` to method names. Just `create`

File: src/seeds/seeder.service.ts [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { SeederRepository } from './seeder.repository';
+import { usersData } from './data/users';
+
+@Injectable()
+export class SeederService {
+  constructor(private readonly seederRepository: SeederRepository) {}
+
+  async startSeedsUsers(): Promise<void> {
```
Comment (boale) [inline src/seeds/seeder.service.ts:9]: it seems that the public method should be `seedData` which will execute several private methods like `seedUsersData`

File: src/seeds/seeder.repository.ts (hunk 7 above)
Comment (boale) [inline src/seeds/seeder.repository.ts:20]: Just make it a private method)

**Review by boale — APPROVED** [review]
Looks like everything is good! Keep it up!

File: src/seeds/seeder.repository.ts (hunk 6 above)
Comment (YuriiBobeshko) [reply src/seeds/seeder.repository.ts]: fixed

**Pull request merged**